sdk/avalon_sdk/work_order_receipt/work_order_receipt.py

```diff
@@ -135,7 +135,7 @@ def update_receipt(self, work_order_id, update_type,
         Returns:
         JSON RPC work order update receipt request of type dictionary
         """
-        data = update_data
+        data = json.dumps(update_data)
```

Comment:
At line 143 also we are converting to serialized string, which is duplicate. instead of line 138 we can use wo_resp_str at 143 and remove serialization at 138.